### ISSUE TITLE
TypeError: AbortSignal.timeout is not a function

### ISSUE THREAD

**Issue by ashemag** [serpapi/serpapi-javascript#4]
Running into this issue with AbortSignal timeout function in typescript: 

```TypeError: AbortSignal.timeout is not a function
    at Object.execute (/Users/ashemagalhaes/Desktop/code/hearth/server/node_modules/serpapi/script/src/utils.js:109:29)
    at getJson (/Users/ashemagalhaes/Desktop/code/hearth/server/node_modules/serpapi/script/src/serpapi.js:61:50)
    at /Users/ashemagalhaes/Desktop/code/hearth/server/src/http/api/controllers/email.ts:196:29
    at Generator.next (<anonymous>)
    at /Users/ashemagalhaes/Desktop/code/hearth/server/src/http/api/controllers/email.ts:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/ashemagalhaes/Desktop/code/hearth/server/src/http/api/controllers/email.ts:4:12)
    at sendAllDailyEmails (/Users/ashemagalhaes/Desktop/code/hearth/server/src/http/api/controllers/email.ts:170:66)
    at Layer.handle [as handle_request] (/Users/ashemagalhaes/Desktop/code/hearth/server/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/ashemagalhaes/Desktop/code/hearth/server/node_modules/express/lib/router/route.js:144:13)```

**Comment by ashemag** [comment]
Resolved by upgrading node above 16.13!

**Comment by pawKer** [comment]
I seem to be getting the same on Node 17.
I'm calling `getJson` like so:

```
const params = {
  q: 'keyword',
  location: 'Greater London,England,United Kingdom',
  gl: 'gb',
  hl: 'en'
}
await getJson("google", params)
```

And I am getting the following error:

```
TypeError: AbortSignal.timeout is not a function
    at Object.execute (file:///<redacted>/node_modules/serpapi/esm/src/utils.js:80:29)
    at getJson (file:///<redacted>/node_modules/serpapi/esm/src/serpapi.js:58:39)
```
